conf/base.config

```diff
@@ -1,63 +1,79 @@
 params {
-  
-  //Default steps
-  runPostQC = true
+
+  /*==========================
+      Which steps to run
+  /*==========================*/
+
+  // Pre-alignment.
+  runFastQC = false
+  runTrimGalore = true
+  runFastQCTrimmed = false
+  runSortMeRNA = true
+
+  // Alignment - STAR
+  runSTAR = true
   runMarkDup = true
-  runHTSeqCount = false
+
+  // Counting - SubRead / FeatureCounts
   runFeatureCounts = true
-  runMapping = true
+
+  // Salmon
   runSalmon = false
-  runTrimGalore = true
-  runSortMeRna = true
+
+  // QC
+  runFastQC = true
+  runFastQCTrimmed = true
   runMultiQC = true
-  runGATK4_HC = false
-  runGATK4_BQSR = false
+  runPostQC = true
+  runPreseq = true
 
-  //Global settings
+  // GATK (v4)
+  runGermlineCallingGATK = false
+  runBQRS = false
+
+  /*==========================
+              Settings
+  /*==========================*/
+
+  // Standard options
   fastq_path = ""
   out_dir = ""
   gencode = false
   singleEnd = false
 
-  //Resources  
+  // Standard references
   genome_config = "$baseDir/conf/test_genome.config"
   rRNA_database_manifest = "$baseDir/assets/sortmerna-db-default.txt"
 
-  //Library strandedness 
+  // Library strandedness
   stranded = false
   revstranded = false
   unstranded = true
 
-  //HTSeq count settings
-  hts_count_type = 'exon'
-  hts_group_features = 'gene_id'
-  count.toolOptions = '-m union -r pos'
+  // Custom settings of tools.
+  options.FastQC = ''
```

Comment:
Kun je dit ook even in de main.nf aanpasen (module includes)? 

Bijvoorbeeld:
params.TrimGalore.toolOptions -> params.options.TrimGalore